README.md

```diff
@@ -136,9 +138,24 @@ For example:
         --accelerator-type ml.eia1.medium \
         --versions 1.13.0
 
+
 ## Pre/Post-Processing
 
-You can add your customized Python code to process your input and output data. To make it work, here are some few things you need to pay attention:
+OutSageMaker TensorFlow Serving Container supports the following Content-Types for requests out of the box:
+
+* `application/json` (default)
+* `text/csv`
+* `application/jsonlines`
+
+And the following Accept types for responses:
+
+* `application/json` (default)
+* `application/jsonlines`
+
+The container will convert data in these formats to [TensorFlow Serving REST API](https://www.tensorflow.org/tfx/serving/api_rest) requests,
+and will send these requests using the default serving signature of your SavedModel.
+
+But you can also add customized Python code to process your input and output data. To make it work, here are some few things you need to pay attention:
 1. The customized Python code file should be named `inference.py` and it should be under `code` directory of your model archive.
```

Comment:
1. add a python file named `inference.py` to the `code` directory inside your model archive.